Controls/Cafe/Customer1.c

```diff
@@ -19,11 +25,29 @@ void Customer1_checkout(char** name, int* cost) {
  */
 void Task_Customer_1(void* p_arg) {
 	OS_ERR err;	// Make sure to check for errors and print the error code if not OS_ERR_NONE
-	
+	CPU_TS ts; 
 	// Use these values as parameters for Customer1_checkout
 	char* name;
 	int cost;
+	CPU_INT16U seconds = 4;//4000ms = 4s DELAY
+	OS_OPT opt =OS_OPT_TIME_HMSM_STRICT;
```

Comment:
You don't need to define these up here if you include the numbers directly in the function call and cast to the required data type